Assets/SteamVR_Unity_Toolkit/Scripts/Helper/DeviceFinder.cs

```diff
@@ -35,9 +35,10 @@ public static SteamVR_TrackedObject TrackedObjectByIndex(uint controllerIndex)
         public static Transform HeadsetTransform()
         {
 #if (UNITY_5_4_OR_NEWER)
-        return GameObject.FindObjectOfType<SteamVR_Camera>().GetComponent<Transform>();
-#endif
+            return GameObject.FindObjectOfType<SteamVR_Camera>().GetComponent<Transform>();
+#else
```

Comment:
this isn't required.

If the first if is successful then the return happens. If it's not successful then the default return happens.

This doesn't fix anything, just does the same thing in a slightly different way.